components/count-badge/count-badge-mixin.js

```diff
@@ -156,25 +157,29 @@ export const CountBadgeMixin = superclass => class extends LocalizeCoreElement(S
 
 	connectedCallback() {
 		super.connectedCallback();
-		if (!this.maxDigits && this.type === 'notification') {
-			// default to two digits for notification type
-			this.maxDigits = 2;
+		if (!this.maxDigits) {
+			// default to two digits for notification type, 5 for count
+			this.maxDigits = this.type === 'notification' ? 2 : 5;
```

Comment:
Looks like it'd be good to save 5 at the top as a constant and then use it in these three places. That way if we ever decide that we want 4 or 6 to be the max digits, we only have to update it in one place.  And then it's obvious that the default for count is actually "max"